lib/Transforms/ConvertToCiphertextSemantics/TypeConversion.cpp

```diff
@@ -72,7 +72,7 @@ Type materializeLayout(RankedTensorType type, LayoutAttr attr,
 
 Type materializeScalarNewLayout(Type type, NewLayoutAttr attr,
                                 int ciphertextSize) {
-  return RankedTensorType::get({ciphertextSize}, type);
+  return RankedTensorType::get({1, ciphertextSize}, type);
```

Comment:
ugh shoot this is fixed in https://github.com/google/heir/pull/2200 but CI was annoying. i'll bump it